python/hail/methods/impex.py

```diff
@@ -900,35 +904,36 @@ def import_table(paths, key=[], min_partitions=None, impute=False, no_header=Fal
         Minimum number of partitions.
 
     no_header: :obj:`bool`
-        File has no header and the N columns are named `f0`, `f1`,
-        ... `fN` (0-indexed).
+        If ``True```, assume the file has no header and name the N columns `f0`,
+        `f1`, ... `fN` (0-indexed).
 
     impute: :obj:`bool`
-        Impute column types from the file.
+        If ``True``, Impute column types from the file.
 
     comment: :obj:`str` or :obj:`None`
-        Skip lines beginning with the given pattern.
+        Skip lines beginning with the given string.
 
     delimiter: :obj:`str`
         Field delimiter regex.
 
     missing: :obj:`str`
-        Specify identifier to be treated as missing
+        Identifier to be treated as missing.
 
     types: :obj:`dict` mapping :obj:`str` to :class:`.Type`
-        Define types of fields in annotations files
+        Dictionary defining column types.
```

Comment:
"field types". When possible, I think we should avoid calling fields columns as it unnecessarily overloads the primary meaning of column in Hail.